### PR TITLE
[PEEPS-2494] PEOPLE: Determine why pubsub messages are not showing up in logs

### PR DESCRIPTION
**Pull request by adbatista**

### CODE REVIEW

File: lib/pub_sub/configuration.rb [hunk 1]
```diff
@@ -1,11 +1,11 @@
 module PubSub
-
   class Configuration
+    attr_reader :logger
     attr_accessor :service_name,
                   :subscriptions,
                   :topics,
                   :visibility_timeout,
-                  :logger,
+                  :log_level,
```
Comment (adamcohen) [inline lib/pub_sub/configuration.rb]: can you add this to the readme to explain how this should be configured?

File: lib/pub_sub/railtie.rb [hunk 2]
```diff
@@ -6,6 +6,7 @@ class Railtie < Rails::Railtie
 
     initializer 'pub_sub.initialize' do
       PubSub.config.logger = Rails.logger
+      PubSub.config.log_level = :debug
```
Comment (adamcohen) [inline lib/pub_sub/railtie.rb]: so does this set the default log level unless overridden in the inititalizer file?

Comment (adbatista) [inline lib/pub_sub/railtie.rb]: yep

**Comment by adamcohen** [comment]
LGTM

**Pull request merged**